dmscripts/send_dos_opportunities_email.py

```diff
@@ -0,0 +1,143 @@
+from datetime import datetime, date, timedelta
```

Comment:
It's possible to avoid importing `date` here by accessing today's date object through `datetime.today().date()` but that's just an FYI not objectively better or worse IMO